keps/349-all-or-nothing/README.md

```diff
@@ -226,6 +232,16 @@ type WaitForPodsReady struct {
 	// RequeuingStrategy defines the strategy for requeuing a Workload.
 	// +optional
 	RequeuingStrategy *RequeuingStrategy `json:"requeuingStrategy,omitempty"`
+
+	// RecoveryTimeout defines an optional timeout, measured since the
+	// last transition to the PodsReady=false condition after a Workload is Admitted and running.
+	// Such a transition may happen when a Pod failed and the replacement Pod 
+	// is awaited to be scheduled.
+	// After exceeding the timeout the corresponding job gets suspended again
+	// and requeued after the backoff delay. The timeout is enforced only if waitForPodsReady.enable=true. 
+	// Defaults to 3 mins.
```

Comment:
nit: I suggest defaulting to timeout, which is 5 mins

Reply:
disregard, it doesn't make sense and we want to have a low default time in case timeout is set to high number.